### PR TITLE
Some block fields

### PR DESCRIPTION
**Pull request by YanisBft**
I think almost all the block fields are now done

### CODE REVIEW

File: mappings/net/minecraft/block/TurtleEggBlock.mapping [hunk 1]
```diff
@@ -1,3 +1,7 @@
 CLASS bnp net/minecraft/block/TurtleEggBlock
+	FIELD a ONE_EGG_SHAPE Lcnl;
+	FIELD b MULTIPLE_EGGS_SHAPE Lcnl;
```
Comment (asiekierka) [inline mappings/net/minecraft/block/TurtleEggBlock.mapping]: ONE_EGG_SHAPE/MULTIPLE_EGG_SHAPE?

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/block/TurtleEggBlock.mapping (hunk 1 above)
Comment (YanisBft) [reply mappings/net/minecraft/block/TurtleEggBlock.mapping]: The block has a small shape when EGGS blockstate value is 1, and a bigger shape when it is 2, 3 or 4

Comment (asiekierka) [reply mappings/net/minecraft/block/TurtleEggBlock.mapping]: Why not just SMALL_SHAPE and BIG_SHAPE?

Comment (YanisBft) [reply mappings/net/minecraft/block/TurtleEggBlock.mapping]: To be more accurate, but could be good too

File: mappings/net/minecraft/world/GameMode.mapping [hunk 2]
```diff
@@ -1,5 +1,9 @@
 CLASS bci net/minecraft/world/GameMode
 	FIELD a INVALID Lbci;
+	FIELD b SURVIVAL Lbci;
```
Comment (asiekierka) [inline mappings/net/minecraft/world/GameMode.mapping]: Didn't these map themselves (in gray)?

**Review by se7kn8 — APPROVED** [review]

File: mappings/net/minecraft/world/GameMode.mapping (hunk 2 above)
Comment (YanisBft) [reply mappings/net/minecraft/world/GameMode.mapping]: They did, I just added them to better see they are mapped

Comment (asiekierka) [reply mappings/net/minecraft/world/GameMode.mapping]: Don't. If they're gray, DON'T touch them.

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**